test/test_translational_input.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
```

Comment:
Test needs to be updated for ROS2, but I could probably chip in with that on a followup PR. Have gotten pretty good at these.

Reply:
Yeah, this was the one bit I hadn't gotten to.  I figured we would do it as a follow-up; would you like me to open an issue about it?

Reply:
Yep, drop us an issue.